src/Persistence/Mapping/Driver/ColocatedMappingDriver.php

```diff
@@ -29,32 +33,45 @@
  */
 trait ColocatedMappingDriver
 {
+    /** @var iterable<string> */
+    protected iterable $sourceFilePathNames;
+
     /**
+     * @deprecated List all needed {@see self::$sourceFilePathNames} explicitly instead.
```

Comment:
Please contribute the deprecations later, as suggested in https://github.com/doctrine/persistence/pull/424#issuecomment-3139639629

For now, let's just treat this as an alternate, equally valid way of using this trait.